Source/Core/Core/IOS/USB/Emulated/Skylanders/SkylanderFigure.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <Common/IOFile.h>
+
+namespace IOS::HLE::USB
+{
+struct SkylanderDateTime final
+{
+  u8 minute;
+  u8 hour;
+  u8 day;
+  u8 month;
+  u16 year;
+};
+
+struct SkylanderData final
+{
+  u16 money;
+  u16 heroLevel;
+  u32 playtime;
+  // Null-terminated UTF-16 string
+  u16 nickname[16];
```

Comment:
`std::array<u16, 16> nickname;`

Reply:
Why should this be a std::array instead of what it is now?